docs/contributing/docs-writing-process.md

```diff
@@ -0,0 +1,99 @@
+
+## Identifying a topic
+
+When identifying a topic, start by:
+
+1. Looking at GitHub issues. Good labels to seek out are [`help wanted`](https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+) and [`good first issue`](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+   - This part may be covered if a docs issue is assigned to you, or if you’re signing up voluntarily to take on an issue.
+
+2. Look at the [stub list](/contributing/stub-list/) to find docs that need contributions.
+
+3. Look at the [learning workflow meta issue](https://github.com/gatsbyjs/gatsby/issues/13708) to find active areas looking for docs.
+
+4. Read through the existing Gatsby docs information and find gaps in topic coverage. Is there an area you feel is missing? [File an issue](/contributing/how-to-file-an-issue/) first to discuss, then implement in a PR.
+
+5. Observe common points of confusion or rough edges through user feedback and recommend solutions.
+
+> _Note: It’s required to open a GitHub issue before submitting a PR if one does not already exist._
+
+## Selecting the correct format
+
+A GitHub issue for new learning material should indicate the format. Is it a Reference or Conceptual Guide? A Tutorial? A recipe?
+
+Does docs coverage exist anywhere on gatsbyjs.org? If so, would an alternative format help provide information for Gatsby learners of different skill and experience levels? For example, if a tutorial exists but there is no coverage in Reference Guides, adding more content in a different format would benefit users.
+
+- Tutorials vs. recipes vs. guides
+  - **Tutorials** are step-by-step learning materials that show a user how to do something without skipping steps. Tutorials must consider a beginner experience, but can be useful for anyone.
+  - **Recipes** are shorter, more concise tutorials that limit their scope to a single task or outcome by using prerequisites and limited steps. Additional resources at the end lead the user to a logical next step. Recipes are for developers of all skill levels.
+  - **Guides** are different than tutorials, and the two main sections are Reference Guides and Conceptual Guides. Instead of “steps 1-4”, guides use standalone headings and sections to explain how to accomplish the task. The audience is often more advanced than beginner due to the subject matter, but should be approachable by anyone.
+
+Given the difference in audiences of the 3 main learning material formats, that’s where increasing and overlapping coverage of topics can help support Gatsby learners of different skill and experience levels.
```

Comment:
```suggestion
Given the difference in audiences of the three main learning material formats, increasing and overlapping coverage of topics can help support Gatsby learners of different skill and experience levels.
```
Do we spell out numbers under ten?

Reply:
We don't have a guideline on it. I typically avoid it when a number leads a sentence or headline, but am fine with them in the text.